pkg/service/backup/restore_model.go

```diff
@@ -108,6 +108,10 @@ func (rp *RestoreRunProgress) setRestoreCompletedAt() {
 	rp.RestoreCompletedAt = &t
 }
 
+func validateTime(t *time.Time) bool {
```

Comment:
`validateTime` against what ?
I see it just checks if time is set.
I propose to change the method signature to include this information.
E.g. **validateTimeIsSet**